src/app/@theme/layouts/one-column/one-column.layout.scss

```diff
@@ -1,9 +1,6 @@
 @import '../../styles/themes';
 @import '~bootstrap/scss/mixins/breakpoints';
 @import '~@nebular/theme/styles/global/breakpoints';
-
 @include nb-install-component() {
-  .menu-sidebar ::ng-deep .scrollable {
-    padding-top: nb-theme(layout-padding-top);
-  }
+
```

Comment:
if you deleted everything, I don't thing you need anything here

Reply:
done